docs/source/core_developers_guide.rst

```diff
@@ -0,0 +1,12 @@
+
+**********************
+Core Developer's Guide
+**********************
+
+.. toctree::
+   :maxdepth: 2
+
+   core_developers_guide/sawtooth_vagrant
+   core_developers_guide/intkey_workload.rst
```

Comment:
Is there a reason this reference uses the `.rst` extension but the other three don't?